PWGMM/Mult/Tasks/dndeta-mft.cxx

```diff
@@ -29,24 +29,7 @@
 #include <CCDB/BasicCCDBManager.h>
 #include <chrono>
 
-namespace o2::aod
-{
-namespace fwdtrack
-{
-DECLARE_SOA_INDEX_COLUMN_FULL(BestCollision, bestCollision, int32_t, Collisions, "");
-DECLARE_SOA_COLUMN(BestDCAXY, bestDCAXY, float);
-DECLARE_SOA_COLUMN(PtStatic, pts, float);
-DECLARE_SOA_COLUMN(PStatic, ps, float);
-DECLARE_SOA_COLUMN(EtaStatic, etas, float);
-DECLARE_SOA_COLUMN(PhiStatic, phis, float);
-} // namespace fwdtrack
-DECLARE_SOA_TABLE(BestCollisionsFwd, "AOD", "BESTCOLLFWD",
-                  aod::fwdtrack::BestCollisionId, aod::fwdtrack::BestDCAXY,
-                  fwdtrack::X, fwdtrack::Y,
-                  fwdtrack::Z, fwdtrack::Tgl, fwdtrack::Signed1Pt,
-                  fwdtrack::PtStatic, fwdtrack::PStatic, fwdtrack::EtaStatic,
-                  fwdtrack::PhiStatic); // Snp does not exist
-} // namespace o2::aod
+#include "bestCollisionTable.h"
```

Comment:
Could you please also make a similar change in dndeta.cxx?

Reply:
Sure !